website/package.json

```diff
@@ -70,5 +74,6 @@
     "react-router": "6.23.1",
     "replace-in-file": "7.2.0",
     "typescript": "4.8.4"
-  }
+  },
+  "packageManager": "yarn@4.2.2"
```

Comment:
Do we want to do `@4` here in case there's a vulnerability and a patch release?

Reply:
It's a fair question, but the regex validator here requires an absolute version. Though, you're allowed to do ^4.2.2 I believe.